tests/fixture_plugins/analysis_starter/store_fixtures.py

```diff
@@ -79,6 +80,14 @@ def mock_housekeeper_for_nf_sample_sheet(fastq_path_1: Path, fastq_path_2: Path)
     return housekeeper_mock
 
 
+@pytest.fixture
+def mock_store_for_nallo_file_creators() -> Store:
+    mock_store: TypedMock = create_typed_mock(Store)
+    mock_store.as_mock.get_case_workflow.return_value = Workflow.NALLO
+    mock_store.as_mock.get_case_priority.return_value = SlurmQos.NORMAL
+    return mock_store.as_type
```

Comment:
```suggestion
    mock_store: Store = create_autospec(Store)
    mock_store.get_case_workflow= Mock(return_value=Workflow.NALLO)
    mock_store.get_case_priority = Mock(return_value=SlurmQos.NORMAL)
    return mock_store
```
But as above works too. Not really certain what the pros and cons are